packages/influx-utilities/package.json

```diff
@@ -0,0 +1,65 @@
+{
+	"name": "@skyra/influx-utilities",
+	"version": "0.0.1",
+	"description": "A tiny opinionated abstraction layer for InfluxDB for interacting with a single organization",
+	"author": "@skyra",
+	"license": "Apache-2.0",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"type": "commonjs",
+	"exports": {
+		"import": "./dist/index.mjs",
+		"require": "./dist/index.js",
+		"types": "./dist/index.d.ts"
+	},
+	"sideEffects": false,
+	"scripts": {
+		"test": "eslint src --ext ts -c ../../package.json",
+		"build": "tsup && tsc -b src",
+		"watch": "tsup --watch --onSuccess \"tsc -b src\"",
+		"watch:types": "tsc -b src -w",
+		"lint": "eslint src --ext ts --fix -c ../../package.json",
+		"prepublish": "rollup-type-bundler -v -e dotenv dotenv-cra",
```

Comment:
rollup-type-bundler is not necessary.

Also the external dependencies would be wrong.